src/components/tabs/__internal__/tab-title/tab-title.spec.tsx

```diff
@@ -16,16 +16,33 @@ import StyledValidationIcon from "../../../../__internal__/validations/validatio
 import Icon from "../../../icon";
 import StyledIcon from "../../../icon/icon.style";
 import Tooltip from "../../../tooltip";
-
-function render(props, renderer = shallow, container = {}) {
+import { ThemeObject } from "../../../../style/themes/base";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
You should move this comment 3 lines below

Reply:
ah yes, thanks - it was correct when I wrote it, but since then Prettier has altered what's on each line!